server/tilesource/base.py

```diff
@@ -340,6 +340,12 @@ def _tileIteratorInfo(self, **kwargs):
             mm_y: the vertical size of a pixel in millimeters.
             exact: if True, only a level that matches exactly will be returned.
                 This is only applied if magnification, mm_x, or mm_y is used.
+        :param tile_position: if present, either a number to only yield the
+            (tile_position)th tile [0 to (xmax - min) * (ymax - ymin)) that the
+            iterator would yield, or a dictionary of {i, j} to yield that tile,
+            where 0, 0 is the first tile yielded, and xmax - xmin - 1,
+            ymax - ymin - 1 is the last tile yielded, or a dictionary of {x, y}
+            to yield that specific tile if it is in the region.
```

Comment:
I would suggest using more meaningful names:

* {tx, ty} instead of {i, j} and 
* {lx, ly} instead of {x, y}